Gruntfile.js

```diff
@@ -47,6 +47,7 @@
                     "generated/idd.heatmapworker_embedded.js",
                     "src/idd/idd.figure.js",
                     "src/idd/idd.chart.js",
+                    "src/idd/idd.uncertainMarkerShapes.js",
```

Comment:
1) idd.uncertainMarkerShapes.js must go after the idd.markers.primitives because this is the file where the shapes should be registered in IDD.Markers.shapes, NOT in the idd.markers.primitives.js

2) rename the file to idd.markers.uncertain.js so we have a kind of guideline of naming files related to markers plot